templates/zerver/portico-header.html

```diff
@@ -45,7 +45,7 @@
             {% endif %}
 
             {% if not is_self_hosting_management_page and not find_team_link_disabled %}
-            <a href="{{ root_domain_url }}/accounts/find/">{{ _('Find accounts') }}</a>
+            <a class= "find-link" href="{{ root_domain_url }}/accounts/find/">{{ _('Find accounts') }}</a>
```

Comment:
Let's go ahead and make this class more expressive: `find-accounts-link` would be much more readable. Be sure to update the CSS selector, too.